frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -154,14 +154,19 @@ const LIGHT_HSL_RANGES = [
   ],
 ];
 
+// We intentionally want to return white text color more frequently
+// https://www.notion.so/Maz-notes-on-viz-settings-67aed0e4ddcc4d4a83028992c4301820?d=513f4f7fa9c143cb874c7e4525dfb1e9#277d6b3eeb464eac86088abd144fde9e
+const whiteTextColorPriorityFactor = 1.8;
+
 export const getTextColorForBackground = (backgroundColor: string) => {
-  const whiteTextContrast = Color(color(backgroundColor)).contrast(
-    Color(color("white")),
-  );
+  const whiteTextContrast =
+    Color(color(backgroundColor)).contrast(Color(color("white"))) *
+    whiteTextColorPriorityFactor;
   const darkTextContrast = Color(color(backgroundColor)).contrast(
     Color(color("text-dark")),
   );
-  return whiteTextContrast > darkTextContrast
+
+  return whiteTextContrast * whiteTextColorPriorityFactor > darkTextContrast
```

Comment:
I'm not sure if this is intended. `whiteTextColorPriorityFactor` is used twice at both line 164 and 169, that effectively equals to
```ts
return oldWhiteTextContrast * whiteTextColorPriorityFactor * whiteTextColorPriorityFactor > darkTextContrst
```

Reply:
@WiNloSt good catch, I moved it and forgot to remove from the return clause